services/scabbard/libscabbard/src/store/mod.rs

```diff
@@ -41,3 +41,5 @@ pub use scabbard_store::PooledPgScabbardStoreFactory;
 pub use scabbard_store::PooledScabbardStoreFactory;
 #[cfg(all(feature = "scabbardv3", feature = "sqlite"))]
 pub use scabbard_store::PooledSqliteScabbardStoreFactory;
+#[cfg(feature = "scabbardv3")]
+pub use scabbard_store::{action, context, service, state, two_phase, ScabbardStore};
```

Comment:
Should we re-export the contents of the other modules? 

(Note - this is a conversation starter, not a request to do anything or blocking the PR)